Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1472,6 +1473,32 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
     }
 }
 
+//
+// Returns component name for eWheelPosition enum
+//
+SString CClientVehicle::GetComponentNameForWheel(unsigned char ucWheel)
+{
+    SString componentName;
+
+    switch (ucWheel)
+    {
+        case FRONT_LEFT_WHEEL:
+            componentName = "wheel_lf_dummy";
+            break;
+        case FRONT_RIGHT_WHEEL:
+            componentName = "wheel_rf_dummy";
+            break;
+        case REAR_LEFT_WHEEL:
+            componentName = "wheel_lb_dummy";
+            break;
+        case REAR_RIGHT_WHEEL:
+            componentName = "wheel_rb_dummy";
+            break;
+    }
+
+    return componentName;
+}
```

Comment:
I really think this shouldn't be in `CClientVehicle`.
Do we have a more generic class for these kinds of utility functions?

Reply:
There are many utility functions that are in this file. So why has no one thought of another class for functions like this before? - In addition, this feature probably won't be used **anywhere** outside of this bugfix.